src/queries/useAuthoritiesDelete/useAuthorityDelete.js

```diff
@@ -0,0 +1,35 @@
+import { useMutation, useQueryClient } from 'react-query';
+import { useOkapiKy, useNamespace } from '@folio/stripes/core';
+
+
+const useAuthorityDelete = ({ onError, onSuccess, ...restOptions }) => {
+  const ky = useOkapiKy();
+  const queryClient = useQueryClient();
+  const [namespace] = useNamespace({ key: 'authoritiesBrowse' });
+
+  const customOptions = {
+    onError: () => {
+      return onError();
+    },
+
+    onSuccess: async () => {
+      await new Promise(resolve => setTimeout(resolve, 500));
```

Comment:
Please add a comment that we need this because `mod-search` requires a little time to update the result list